app/src/test/report/components/Artefacts/ArtefactsListTests.tsx

```diff
@@ -6,25 +6,31 @@ import {mockArtefact} from "../../../mocks/mockModels";
 
 describe("ArtefactsList", () => {
     it("renders list of ArtefactItems", () => {
-
         const fakeArtefactsArray = [
             {
-                "staticgraph": mockArtefact({filename: "file.csv", description: "a file"})
+                "staticgraph": mockArtefact({
```

Comment:
If there's any way of harmonising the test data orderly produces and the information here, that might help keep things in sync more easily